src/domUpdates.js

```diff
@@ -62,10 +62,26 @@ const renderGrid = () => {
   recipeGrid.innerHTML = createGridHTML(gridData);
 }
 
+const showRecipe = (recipeCard) => {
```

Comment:
NICELY DONE!
What do you think about letting event passed in as an argument all the way to this function and THEN make the recipeCard? I'm not too sure (and keen to hear from you) but I somehow feel it's good practice to do so?

Reply:
maybe this was confusing, `recipeCard` IS the `event.target` from `scripts.js`!  Want me to just pass in the `event` and name the parameter `event` here?

Reply:
Oh yes I understand that. I suppose I was just wondering if event being passed was better than event.target. 
I sincerely do not know and was just wondering. I'm sure either is fine so it does not need to change.

Reply:
I think it is ok to pass event.target and have the parameter semantically named

Reply:
Makes sense. Can definitely keep it as is.